src/GeneralStateTestsFiller/stEWASMTests/callValueExceedsBalanceFiller.yml

```diff
@@ -0,0 +1,177 @@
+              (get_local $ptrCallData)
+              ;; data offset
+              (i32.const 0)
+              ;; length
+              (get_local $callDataSize)
+            )
+
+            ;; store call data
+            (call $storageStore
+              (get_local $ptrStorageKey11)
+              (get_local $ptrCallData)
+            )
+          )
+        )
+    deadbeef00000000000000000000000000000000:
+      balance: 0
+      nonce: ''
+      storage: {}
+      code: |
+        (module
+          (import "ethereum" "call" (func $call (param i64 i32 i32 i32 i32) (result i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+
+          (memory 1)
+          (data (i32.const 0) "\01\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\ef\be\ad\de")
+
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+
+          (func $main
+            ;; locals
+            (local $memStart i32)
+            (local $ptrStorageKey1 i32)
+            (local $ptrValue i32)
+            (local $ptrData i32)
+            (local $ptrResult i32)
+
+            ;; init
+            (set_local $memStart (i32.const 32))
+
+            ;; init pointers
+            (set_local $ptrStorageKey1 (get_local $memStart))
+            (set_local $ptrValue (i32.add (get_local $memStart) (i32.const 32)))
+            (set_local $ptrData (i32.add (get_local $memStart) (i32.const 64)))
+            (set_local $ptrResult (i32.add (get_local $memStart) (i32.const 96)))
+
+            ;; init mem
+            (i32.store (get_local $ptrStorageKey1) (i32.const 1))
+            ;; value is 1 greater than this account's balance
+            (i32.store (get_local $ptrValue) (i32.const 1))
+            (i32.store (get_local $ptrData) (i32.const 0))
+
+            ;; call CALL with data and save result
+            (i32.store
+              (get_local $ptrResult)
+              (call $call
+                ;; gas limit
+                (i64.const 200000)
+                ;; address offset (stored in data section above)
+                (i32.const 0)
+                ;; value offset
+                (get_local $ptrValue)
+                ;; data offset
+                (get_local $ptrData)
+                ;; data length
+                (i32.const 0)
+              )
+            )
+
+            ;; store the result
+            (call $storageStore
+              (get_local $ptrStorageKey1)
+              (get_local $ptrResult)
+            )
+          )
+        )
+  expect:
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value: !!int -1
+      network:
+        - ALL
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          # Charged for call but unused gas refunded
+          balance: 99999951600
+        deadbeef00000000000000000000000000000000:
+          # no change in balance
+          balance: 0
+          storage: {
+            # call should have failed
+            1: 1
```

Comment:
This test is failing for me, I'm getting 0 for key 1. Could you share which version of cpp and hera are you using?